blueprints/ember-cli-head/index.js

```diff
@@ -1,7 +1,27 @@
 /*jshint node:true*/
+let fs = require('fs');
+let path = require('path');
+let chalk = require('chalk');
+
 module.exports = {
   description: '',
-  normalizeEntityName: function() { }
+  normalizeEntityName: function() { },
+  afterInstall() {
+    let fullPath = path.join(this.project.root, 'app/templates/applicati.hbs');
+
+    if (fs.existsSync(fullPath)) {
+      let contents = fs.readFileSync(fullPath, { encoding: 'utf-8' });
+      let toWrite = `{{head-layout}}\n${contents}`;
+
+      fs.writeFileSync(fullPath, toWrite, { encoding: 'utf-8' });
+    } else {
+      let str = `You must add {{head-layout}} component to your topmost UI.
+This is usually your application.hbs, but was not found on your system.
```

Comment:
Maybe use the full path (e.g. `app/templates/application.hbs`) in the message?

Reply:
Updated!  🍻